### DISCUSSION TITLE
Hello teachers! After updating, an error was reported. How should this error be resolved? Please advise!

### DISCUSSION THREAD

**Discussion by zjgzh001** [azerothcore/azerothcore-wotlk#16459] [General]
Hello teachers! After updating, an error was reported. How should this error be resolved? Please advise!
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/133203282/2d0998fe-e10e-4157-ae85-65beedd4297e)

**Comment by Nyeriah** [comment]
Always compile your core after updating.
